src/pages/ReimbursementAccount/BeneficialOwnersStep.js

```diff
@@ -106,6 +133,26 @@ class BeneficialOwnersStep extends React.Component {
             || (_.size(this.state.beneficialOwners) === 3 && !this.state.ownsMoreThan25Percent);
     }
 
+    /**
+     * Clear the error associated to inputKey if found and store the inputKey new value in the state.
+     *
+     * @param {Integer} ownerIndex
+     * @param {String} inputKey
+     * @param {String} value
+     */
+    clearErrorAndSetBeneficialOwnerValue(ownerIndex, inputKey, value) {
+        this.setState((prevState) => {
+            const beneficialOwners = [...prevState.beneficialOwners];
+            beneficialOwners[ownerIndex] = {...beneficialOwners[ownerIndex], [inputKey]: value};
+            updateReimbursementAccountDraft({beneficialOwners});
+            return {beneficialOwners};
+        });
+        if (inputKey === 'dob') {
+            this.clearError(`beneficialOwnersErrors.${ownerIndex}.dobAge`);
```

Comment:
NAB, I think this is OK for now. But we're going to have to figure out a better way to handle individual fields with multiple possible errors in the future.